pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1008,11 +1055,39 @@ func createTapDevice() (string, error) {
 		log.Log.Reason(err).Criticalf("Failed to create tap device %s. Reason: %s", tapName, out)
 		return "", err
 	}
+	log.Log.Infof("Created tap device: %s", tapName)
 	return tapName, nil
 }
 
 func generateTapDeviceName() string {
 	generatedUUID := uuid.New().String()
 	generatedUUID = strings.ReplaceAll(generatedUUID, "-", "")
 	return fmt.Sprintf("tap%s", generatedUUID[0:12])
-}
\ No newline at end of file
+}
+
+func configureTapDevice(tapName string, bridgeName string) error {
+	tap, err := netlink.LinkByName(tapName)
+	log.Log.Infof("Looking for tap device: %s", tapName)
+	if err != nil {
+		log.Log.Infof("DID NOT FIND tap device: %s", tapName)
```

Comment:
Why not `log.Log.Reason(err).Errorf`?

Reply:
No good reason; that is the proper way; thanks for pointing it out.